fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java

```diff
@@ -133,6 +135,19 @@ public QueryQueue getWorkloadGroupQueryQueue(String groupName) throws UserExcept
         }
     }
 
+    private String getWorkloadGroupName(ConnectContext context) throws AnalysisException {
+        String groupName = context.getSessionVariable().getWorkloadGroup();
+        if (Strings.isNullOrEmpty(groupName)) {
+            groupName = Env.getCurrentEnv().getAuth().getWorkloadGroup(context.getQualifiedUser());
+        }
+        if (!Env.getCurrentEnv().getAccessManager().checkWorkloadGroupPriv(context, groupName, PrivPredicate.USAGE)) {
```

Comment:
If `groupName` is still null, set it to null?

Reply:
The `default_workload_group` in the user property is `normal` by default